llap-server/src/java/org/apache/hadoop/hive/llap/daemon/impl/ContainerRunnerImpl.java

```diff
@@ -649,7 +649,7 @@ static class UgiPool {
           @Override
           public void onRemoval(
               RemovalNotification<String, BlockingQueue<UserGroupInformation>> notification) {
-            LOG.debug("Removing " + notification.getValue()  + " from pool.Pool size: " + ugiPool.size());
+            LOG.debug("Removing {} from Pool size{}: ", notification.getValue(), ugiPool.size());
```

Comment:
Missing "pool." and the value of size should be after ":" . Like below

"Removing {} from pool. Pool size:{}

Reply:
done